plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/variations/elements/product-title.tsx

```diff
@@ -3,25 +3,25 @@
  */
 import { Icon } from '@wordpress/components';
 import { heading } from '@wordpress/icons';
+import {
+	title,
+	description,
+} from '@woocommerce/atomic-blocks/product-elements/title/block.json';
```

Comment:
This is a confusing alias IMO, given it existed for a while I did be ok with this. But now that blocks lives within the monorepo we should remove this, or move `atomic-blocks` to a folder in the `packages/js` list.
If we do keep it within, I would be tempted to support something like this `~` which would be the root of the blocks directory.

Reply:
I don't have a strong opinion on that but I see your points and tend to agree. However, for now I prefer to keep it consistent with the other parts and change it globally.